app/models/student.rb

```diff
@@ -39,4 +63,19 @@ class Student < ActiveRecord::Base
   has_many :teachers, through: :matchings
   has_many :instruments, as: :instrumentable
 
+  enum school_level: [ :kindergarten, :'1st grade', :'2nd grade', :'3rd grade',
+                      :'4th grade', :'5th grade', :'6th grade', :'7th grade',
+                      :'8th grade', :'9th grade', :'10th grade', :'11th grade',
+                      :'12th grade' ]
+  enum state: [ :AL, :AK, :AZ, :AR, :CA, :CO, :CT, :DE, :FL, :GA,
+                :HI, :ID, :IL, :IN, :IA, :KS, :KY, :LA, :ME, :MD,
+                :MA, :MI, :MN, :MS, :MO, :MT, :NE, :NV, :NH, :NJ,
+                :NM, :NY, :NC, :ND, :OH, :OK, :OR, :PA, :RI, :SC,
+                :SD, :TN, :TX, :UT, :VT, :VA, :WA, :WV, :WI, :WY ]
+  enum gender: [ :female, :male, :other ]
+  enum travel_distance: [ :'I am not willing to travel', :'Up to 5 miles', 
+                          :'Up to 10 miles', :'Up to 20 miles', 
+                          :'20 miles or more']
+  enum income_range: [:'$0 - $10,000', :'$10,001 - $20,000', :'$20,001 - 30,000', :'$30,001 - 40,000', :'$40,001 - $50,000']
+
```

Comment:
should we add a note here and in the javascript constants that if you change one of these you should change both